README.md

```diff
@@ -79,6 +79,16 @@ Then get the JS files following the instructions below.
   then only the specified locales will be exported.
   Current version of `i18n.js` will also be exported to avoid version mismatching by downloading.
 
+#### Rails with [webpacker](https://github.com/rails/webpacker)
+
+If you're using `webpacker`, you may need to add the dependencies to your client with:
+
+```
+yarn add i18n-js
```

Comment:
```shell
# Depends on your package manager
npm install i18n-js
# or
yarn add i18n-js
```

Reply:
Okay! I chose `yarn` since it's what `webpacker` uses and you have to go through some hoops to use `npm` with `webpacker` but this sounds good!

Reply:
dbab13a